gulpfile.babel.js

```diff
@@ -34,7 +36,13 @@ gulp.task('lint', () => {
     .pipe(plugins.eslint.format());
 });
 
-gulp.task('test', ['lint']);
+gulp.task('mocha-test', () => {
+  return gulp.src(yargs.argv.file ? yargs.argv.file : 'test/test*.js', {read: false})
```

Comment:
- `file` as argument isn't very descriptive. `test` or `suite` might fit its single use case better.
- use `yargs.option` to set a default for file, like `const argv = yargs.option('s', {alias: 'suite', default: 'test/test*.js'})`
- I am not a big enthusiast of naming files in a `test` folder `testSomething` but rather stick to `test/engine.js` to keep it (KI)SS. I am open for arguments against this simplification though.